docs/src/pages/guides/aliases.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Aliases
+---
+
+Aliasing is an integral part when it comes to clean and readable code, that's why Astro comes with support for custom aliases out of the box.
+
+## What are aliases?
+
+Aliases are _keywords_ you can use instead of paths. 
+So instead of `../../components/SomeComponent/` you can use `components/SomeComponent`. 
+
+## Adding a custom alias
+
+If you navigate to the root directory of your project and open the `snowpack.config.mjs` file you will notice that there are some pre-defined aliases already.
+
+```ts
+// snowpack.config.mjs
+
+alias: {
```

Comment:
since some people copy-paste these directly, I'd use the full file format (at least for the first example):

```
// snowpack.config.mjs
export default {
```